pkg/reconciler/kubernetes/tektonconfig/controller.go

```diff
@@ -28,7 +28,11 @@ import (
 
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
 	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
+	tektonAddoninformer "github.com/tektoncd/operator/pkg/client/injection/informers/operator/v1alpha1/tektonaddon"
 	tektonConfiginformer "github.com/tektoncd/operator/pkg/client/injection/informers/operator/v1alpha1/tektonconfig"
+	tektonDashboardinformer "github.com/tektoncd/operator/pkg/client/injection/informers/operator/v1alpha1/tektondashboard"
```

Comment:
similar to above this can be moved to https://github.com/tektoncd/operator/blob/7490fc2095c004f17c1946175682cb938b7efb18/pkg/reconciler/kubernetes/tektonconfig/controller.go#L44

Reply:
How this can be openshift specific ?? 🤔 
Deleting a component on any k8s should get recreated right ??

Reply:
it is k8s specific

Reply:
we don't have dashboard for openshift

Reply:
OhhhI got it now thanks 👍🏻